ios/FluentUI/Drawer/DrawerShadowView.swift

```diff
@@ -9,7 +9,22 @@ class DrawerShadowView: UIView {
     private struct Constants {
         static let shadowRadius: CGFloat = 4
         static let shadowOffset: CGFloat = 2
-        static let shadowOpacity: Float = 0.05
+        static let shadowOpacity: Float = 0.1
```

Comment:
Is there any design guidance on what the value of the opacity should be?
It's good to check with them if we're changing the value.

Reply:
I got go-ahead from design for it.

Reply:
This change got reverted!